Packages/packages-lock.json

```diff
@@ -19,7 +19,7 @@
       "depth": 0,
       "source": "registry",
       "dependencies": {
-        "com.unity.test-framework": "1.1.1"
+        "com.unity.test-framework": "1.1.3"
```

Comment:
Make sure you pull from master so you have the updated .gitignore That would prevent this file from showing up